CHANGELOG.md

```diff
@@ -3,20 +3,31 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.51.3
```

Comment:
This is a minor release – patch releases should only be for tiny changes (essentially one line bug fixes really).